demos/modipulate-gml/test/main.c

```diff
@@ -0,0 +1,200 @@
+/* Demo test app for modipulate-gml */
+
+#include <stdio.h>
+#include "modipulate_gml.h"
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
Yep, looks like various OpenMPT files do, as well as the console demo and modipulate-lua. I can find all those after fixing this branch.